src/Mod/Draft/draftobjects/patharray.py

```diff
@@ -662,7 +662,7 @@ def placements_on_path(shapeRotation, pathwire, count, xlate, align,
             _wrn(
                 translate(
                     "draft",
-                    "End Offset too large for path length minus Start Offset. Using zero instead."
+                    "The end offset is too large for  the path length minus the start offset. Using 0 instead."
```

Comment:
Properties are capitalized in the GUI. IMO we should follow that here. Lines 655 and 665 should be more consistent. Why add "the" for only one warning in this file?